onnxruntime/core/providers/cuda/math/matmul_integer.cc

```diff
@@ -106,6 +107,28 @@ Status MatMulInteger<int8_t, int8_t>::ComputeInternal(OpKernelContext* ctx) cons
     beta = 1;
   }
 
+#if CUDA_VERSION >= 10010
+  if (DeviceProp::GetDeviceProps().major >= 7 && DeviceProp::GetDeviceProps().minor >= 5) {
```

Comment:
GPU | Compute Capability
-- | --
Tesla T4 | 7.5
Tesla V100 | 7.0

Since V100 also has Tensor Core. I think we need not check the minor.

Reply:
Volta's tensor cores only supports FP16. Turing tensor cores extend the support to int8 and int4.

---
In reply to: [404454273](https://github.com/microsoft/onnxruntime/pull/3413#discussion_r404454273) [](ancestors = 404454273)